integration-tests/integration/receptionMove/receptionConsiderRisksPage.cy.js

```diff
@@ -0,0 +1,362 @@
+        offenderNo: 'G0873UU',
+        classificationCode: 'HI',
+        classification: 'High',
+        assessmentCode: 'CSR',
+        assessmentDescription: 'CSR Rating',
+        cellSharingAlertFlag: true,
+        assessmentDate: '2016-12-22',
+        nextReviewDate: '2016-12-23',
+        assessmentAgencyId: 'MDI',
+        assessmentStatus: 'A',
+        assessmentSeq: 1,
+        assessmentComment: "Don't put in cell with others",
+        assessorId: 44495,
+        assessorUser: 'VQT41X',
+      },
+      {
+        bookingId: 1161007,
+        offenderNo: 'G6795VD',
+        classificationCode: 'HI',
+        classification: 'Low',
+        assessmentCode: 'CSR',
+        assessmentDescription: 'CSR Rating',
+        cellSharingAlertFlag: true,
+        assessmentDate: '2016-12-22',
+        nextReviewDate: '2016-12-23',
+        assessmentAgencyId: 'MDI',
+        assessmentStatus: 'A',
+        assessmentSeq: 1,
+        assessmentComment: "Don't put in cell with others",
+        assessorId: 44495,
+        assessorUser: 'VQT41X',
+      },
+    ],
+  })
+  cy.task('stubCellMoveTypes', [
+    {
+      domain: 'CHG_HOUS_RSN',
+      code: 'RAIM',
+      description: 'Reception and induction moves',
+      activeFlag: 'Y',
+      listSeq: 1,
+      systemDataFlag: 'N',
+      subCodes: [],
+    },
+    {
+      domain: 'CHG_HOUS_RSN',
+      code: 'SS',
+      description: 'Someone’s safety',
+      activeFlag: 'Y',
+      listSeq: 2,
+      systemDataFlag: 'N',
+      subCodes: [],
+    },
+    {
+      domain: 'CHG_HOUS_RSN',
+      code: 'SPP',
+      description: 'Security of the prison or other people',
+      activeFlag: 'Y',
+      listSeq: 2,
+      systemDataFlag: 'N',
+      subCodes: [],
+    },
+    {
+      domain: 'CHG_HOUS_RSN',
+      code: 'HOSP',
+      description: 'Healthcare',
+      activeFlag: 'Y',
+      listSeq: 2,
+      systemDataFlag: 'N',
+      subCodes: [],
+    },
+    {
+      domain: 'CHG_HOUS_RSN',
+      code: 'PCM',
+      description: 'Maintenance of the prison or cell',
+      activeFlag: 'Y',
+      listSeq: 2,
+      systemDataFlag: 'N',
+      subCodes: [],
+    },
+    {
+      domain: 'CHG_HOUS_RSN',
+      code: 'PCGMM',
+      description: 'General moves',
+      activeFlag: 'Y',
+      listSeq: 2,
+      systemDataFlag: 'N',
+      subCodes: [],
+    },
+  ])
+  cy.task('stubOffenderNonAssociationsLegacy', {
+    agencyDescription: 'HMP Moorland',
+    offenderNo: 'G3878UK',
+    nonAssociations: [
+      {
+        effectiveDate: moment(),
+        expiryDate: moment().add(2, 'days'),
+        offenderNonAssociation: {
+          agencyDescription: 'HMP Moorland',
+          assignedLivingUnitDescription: 'HMP Moorland',
+          offenderNo: 'G3878UK',
+          firstName: 'UIENCOT',
+          lastName: 'BLANCYANA',
+          agencyId: 'MDI',
+        },
+      },
+    ],
+  })
+  cy.task('stubOffendersInReception', {
+    agencyId: 'MDI',
+    inReception: [
+      {
```

Comment:
I don't think we use a lot of the data in each prisoners object here. Can it be limited to just the bits used in the code (i.e name, offenderNo, alerts). This is just so the test files aren't unnecessarily large